Library/Formula/xxhash.rb

```diff
@@ -0,0 +1,16 @@
+class Xxhash < Formula
+  desc "Extremely fast non-cryptographic hash algorithm"
+  homepage "https://github.com/Cyan4973/xxHash"
+  version "r42"
+  url "https://github.com/Cyan4973/xxHash/archive/r42.tar.gz"
+  sha256 "d21dba3ebf5ea8bf2f587150230189231c8d47e8b63c865c585b08d14c8218b8"
+
+  def install
+    system "make"
+    bin.install "xxhsum"
+  end
+
+  test do
+    system "#{bin}/xxhsum", "#{bin}/xxhsum"
```

Comment:
Presuming it hashes consistently, it may be nice to do something like:

```ruby
test do
  (testpath/"hash.txt").write "Something to hash"
  assert_match /expected_hash_here/, shell_output("#{bin}/xxhsum hash.txt")
end
```